README.md

```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Name
+
+Hello, my name is **Nadiah Wong**! 
+
+
+## Who am I?
+
+I was born on 6 July 1997 in sunny Singapore
+This is what I look like: ![*Premature graduation photos because we probably would not be back together again soon :-(*](/Users/nadiah/Desktop/grad photos/DSC03848-2)
+
+
+### Interests
+1. Party Tricks
+    + I can do a headstand and fold my tongue to look like a three leaf clover
+2. Hobbies
+    + I love to play the guitar
+    + Photography
+    + Acting
```

Comment:
For lists, you can also use bullet points:
* Party Tricks
   + I can ....

Reply:
And for enumerated lists, you don't even have to number the elements yourself. Markdown does it automatically

```
1. Party Tricks
    + I can do a headstand and fold my tongue to look like a three leaf clover
1. Hobbies
    + I love to play the guitar
    + Photography
    + Acting
```

1. Party Tricks
    + I can do a headstand and fold my tongue to look like a three leaf clover
1. Hobbies
    + I love to play the guitar
    + Photography
    + Acting